cognite/powerops/client/shop/data_classes/workflow.py

```diff
@@ -0,0 +1,25 @@
+from pathlib import Path
+from typing import Literal
+
+from pydantic import BaseModel, field_validator
+
+class ShopCase(BaseModel):
+    pre_run_file: Path | str
+    plants: list[str]
+    price_scenario: str
+
+class Case(BaseModel):
+    case_name: str # e.g. Glomma
+    plants: list[str]
+    pre_runs: list[ShopCase]
+
+class Workflow(BaseModel):
```

Comment:
I think workflow is an unfortunate name as it matched the `python-sdk` name. But I do not have a better alternative at the top of my head.

Reply:
or PoweropsWorkflow? In case we want to extend this to run a workflow for other markets?